satpy/readers/modis_l1b.py

```diff
@@ -15,13 +15,34 @@
 #
 # You should have received a copy of the GNU General Public License along with
 # satpy.  If not, see <http://www.gnu.org/licenses/>.
-"""Interface to Modis level 1b format.
-http://www.icare.univ-lille1.fr/wiki/index.php/MODIS_geolocation
-http://www.sciencedirect.com/science?_ob=MiamiImageURL&_imagekey=B6V6V-4700BJP-\
-3-27&_cdi=5824&_user=671124&_check=y&_orig=search&_coverDate=11%2F30%2F2002&vie\
-w=c&wchp=dGLzVlz-zSkWz&md5=bac5bc7a4f08007722ae793954f1dd63&ie=/sdarticle.pdf
-"""
 
+"""Modis level 1b hdf-eos format reader
+
+Introduction
+------------
+
+The ``modis_l1b`` reader reads and calibrates Modis L1 image data in hdf-eos format. Files often have
+a pattern similar to the following one:
+
+.. parsed-literal::
+    M[O/Y]D02[1/H/Q]KM.A[date].[time].[collection].[processing_time].hdf
```

Comment:
Does this render properly? Or does it need an extra line between the parsed-literal and the pattern?

Reply:
http://rst.ninjs.org/ doesn't like the parsed-literal instruction...

Reply:
@mraspaud That may be a sphinx-only directive. Or it may need the extra line.

Reply:
https://livesphinx.herokuapp.com/ Seems to be ok with no extra line and with an extra line.